rancher2/schema_global_role.go

```diff
@@ -0,0 +1,49 @@
+package rancher2
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+//Schemas
+
+func globalRoleFields() map[string]*schema.Schema {
```

Comment:
As mentioned before, `globalRoleFields()` should be different from dataSource and Resource. Could you please add another function to define dataSource flelds (here or in `rancher2/data_source_rancher2_global_role.go` file)??

Something like,
```
func dataSourceGlobalRoleFields() map[string]*schema.Schema {
	s := map[string]*schema.Schema{
		"builtin": {
			Type:        schema.TypeBool,
			Computed:    true,
			Description: "Builtin global role",
		},
		"description": {
			Type:        schema.TypeString,
			Computed:    true,
			Description: "Global role policy description",
		},
		"name": {
			Type:        schema.TypeString,
			Required:    true,
			Description: "Global role policy name",
		},
		"new_user_default": {
			Type:        schema.TypeBool,
			Computed:    true,
			Description: "Whether or not this role should be added to new users",
		},
		"rules": {
			Type:        schema.TypeList,
			Computed:    true,
			Description: "Global role policy rules",
			Elem: &schema.Resource{
				Schema: policyRuleFields(),
			},
		},
                "annotations": {
	  	        Type:     schema.TypeMap,
			Computed: true,
		},
		"labels": {
			Type:     schema.TypeMap,
			Computed: true,
		},
	}

	return s
}
```